next/components/forms/widgets/ExampleInput.tsx

```diff
@@ -10,6 +13,15 @@ export const ExampleInput = (props: any) => {
         required={props.required}
         onChange={(event) => props.onChange(event.target.value)}
       />
+      <Button variant='link-black' href='/stackoverflow.com/' label='Link value'></Button>
```

Comment:
Empty components should be self-closing (eslint should complain about this).

```suggestion
      <Button variant='link-black' href='/stackoverflow.com/' label='Link value' />
```